AdvLoggerPkg/Library/DebugAgent/Sec/AdvancedLoggerSecDebugAgent.inf

```diff
@@ -22,16 +22,19 @@
 #
 # The following information is for reference only and not required by the build tools.
 #
-#  VALID_ARCHITECTURES           = IA32
+#  VALID_ARCHITECTURES           = IA32 X64
 #
 
 [Sources]
   AdvancedLoggerSecDebugAgent.c
+  RamForSEC.c
 
-[Sources.Ia32]
-  ia32/RamForSEC.c
+[Sources.IA32]
   ia32/AsmRepLodsd.nasm

```

Comment:
I wanted to change this to `IA32` instead of `ia32` but dealing with some of nuances of only changing case in git wasn't worth it.